addons/mail/static/src/discuss/call/common/call_action_list.js

```diff
@@ -52,4 +61,17 @@ export class CallActionList extends Component {
     async onClickToggleAudioCall(ev, { camera = false } = {}) {
         await this.rtc.toggleCall(this.props.thread, { camera, fullscreen: this.props.fullscreen });
     }
+
+    onEnterMore() {
+        if (this.props.thread.videoCount > 0 && this.env.inChatWindow) {
+            this.popover.open(this.more.el, { tooltip: _t("Enter full screen!") });
```

Comment:
since popovers are in global components, they aren't technically children of the call view so the it can cause a flicker if the mouse enters that popover:

![CleanShot 2025-04-29 at 10 47 22](https://github.com/user-attachments/assets/1f62f26f-4198-4dce-b2f8-11f48369e996)


you can prevent it by adding a class to the popover and exclude it here: https://github.com/odoo/odoo/blob/master/addons/mail/static/src/discuss/call/common/call.js#L204